bin/BuildPackages.sh

```diff
@@ -321,7 +322,7 @@ do
   if [ "x$PARALLEL" = "xyes" ]; then
     # If more than 4 background jobs are running, wait for one to finish (if
     # <wait -n> is available) or for all to finish (if only <wait> is available)
-    if [[ $(jobs -r -p | wc -l) -gt 4 ]]; then
+    if [[ $(jobs -r -p | wc -l) -gt $((NCORE +1)) ]]; then
```

Comment:
Note that by telling both `make` and `jobs` to use that many cores, you are in fact instructing the system to use ~NCORE^2 cores.

One could possibly rearrange things so that a single `make` job server is shared across all jobs, then it's be "only" 2*NCORE jobs.

Reply:
This needs to tweak the script `bin/BuildPackages.sh` a lot. If you have a more feasible implementation, I will be glad to see your improved code.

Reply:
Nope, I don't -- there's a reason why we implemented the script the way it is right now: it's a compromise between being portable and still using parallelism *somewhat*. In practice I can build all packages in under 2 minutes with this, an operation most people only perform once a year, if at all -- so this seems like an adequate compromise.

If you feel you need to run this often enough so that the time you spend on improving the script merits the effort, then I think the best way to go about it is to do what I suggested above, and add a command line option to the script that allows overriding these limits.